_pages/meetings.md

```diff
@@ -51,6 +51,8 @@ _pages/meetings/april2020.md %}), April 29 (remote).
 
 #### AGU Sessions
 
-* AGU Fall 2021 meeting: SH023 - Python Applications for Solar and Space Physics (virtual-only poster session) 
-    * [Registration](https://www.agu.org/Fall-Meeting/Pages/Register-Housing) opens 23 August 2021 
+* AGU Fall 2021 meeting: SH35H - Python Applications for Solar and Space Physics I Poster (virtual-only poster session) 
```

Comment:
I can certainly change that - it was listed that way as that's how it's listed on the link I give for our poster session.

Reply:
Yeah let's change it. Idk why that was listed that way but I'm almost certain it's "iPoster" because AGU uses the  "aMuze! iPoster platform" for its posters.

Reply:
Makes sense. Thanks for making the change! I'll merge now.